README.rst

```diff
@@ -770,16 +779,16 @@ to install using pip and a virtualenv::
 
     virtualenv env
     source env/bin/activate
-    python synapse/python_dependencies.py | xargs -n1 pip install
-    pip install setuptools_trial mock
+    python synapse/python_dependencies.py | xargs pip install
+    pip install lxml mock
```

Comment:
well, this is why I added the "make sure you have the system dependencies installed" paragraph. We might as well duplicate the whole section, and I'm super unkeen on that.